<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgMultiTempData</class>
 <widget class="QDialog" name="DlgMultiTempData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>218</width>
    <height>141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Сравнваемое изображение</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="EditObzor"/>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>Обзор</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Левый порог</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="EditLeftPor"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Правый порог</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="EditRigthPor"/>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="ButtonOK">
     <property name="text">
      <string>ОК</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QPushButton" name="ButtonCancel">
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
